### ISSUE TITLE
Patch file is not getting created

### ISSUE THREAD

**Issue by Mayank-7** [insidesherpa/bank-mergeCoTechTask1#92]
followed all the steps to create the patch file but no patch file is getting created. even zip [doesn't]
![Screenshot (86)](https://user-images.githubusercontent.com/52287975/81004482-15b3bd80-8e6a-11ea-89ac-ba74b2c2fc52.png)
 contain the patch file.

**Comment by lionelsamrat10** [comment]
Same issue buddy

**Comment by mayurisannap** [comment]
Same for me too. Have you found any solution to this?

**Comment by lionelsamrat10** [comment]
After running all the git commands properly just refresh the page.. The patch file will be there..

**Comment by Mayank-7** [comment]
Yes, I got a solution for it. You just have to register into repl.it with your github account and then perform all the steps. Your Patch file will bee created

**Comment by mayurisannap** [comment]
Yeah, I did the changes after registering only, but still the .patch file wasn't visible in list. But refreshing the page helped! Thank you @lionelsamrat10 and @Mayank-7

**Comment by Syeda-Sufia** [comment]
Use double quotes in "Create Patch File"

**Comment by IFHAM99** [comment]
The patch file got created after refreshing but after downloading and extracting, I am not able to find any .patch file

**Comment by snehalkhandve** [comment]
Refreshing the page has worked for me too !! Thanks

**Comment by snehalkhandve** [comment]
> Refreshing the page has worked for me too !! Thanks

But on downloading the folder patch file is not present
